doc/catalog-search-oai-harvest-process.md

```diff
@@ -0,0 +1,35 @@
+Catalog Search OAI Harvest Process
+===
+
+## Summary
+The current catalog OAI harvesting process is broken up into two distinct but linked processes and their respective DAGs:
+* [The Pre Production Catalog OAI Harvest](../cob_datapipeline/prod_catalog_oai_harvest_dag.py)
+* [The Production Catalog OAI Harvest](../cob_datapipeline/catalog_preproduction_oai_harvest_dag.py)
+
+Each of these DAGs will harvest independently to a configured collection:
+* `catalog_pre_production_oai_harvest`  harvests:
+    * from date set in `CATALOG_PRE_PRODUCTION_SOLR_COLLECTION` variable.
+    * until `Now()` by default.
+    * into the collection set in `CATALOG_PRE_PRODUCTION_HARVEST_FROM_DATE` variable.
+
+* `catalog_preproduction_oai_harvest_dag` harvests:
+    * from date set in `CATALOG_PROD_HARVEST_FROM_DATE` variable.
+    * until `Now()` by default.
+    * into the collection set in `CATALOG_PRODUCTION_SOLR_COLLECTION` variable
+
+## Collection swap process.
+When tul_cob gets deployed, the `CATALOG_PRODUCTION_SOLR_COLLECTION` will get
```

Comment:
You should be more specific: this only happens when a tul_cob release is deployed to production.